R/app_server.R

```diff
@@ -52,7 +79,8 @@
   # Save the user details
   user_info <- shiny::reactive({
     data.frame(
-      user_name = input$username,
+      user_name = slack_user_info()$user_name,
```

Comment:
We probably just need user-id to be saved in the googlesheet/database. The username can just be used to display.

Reply:
It made it confusing to review the results as a human. We can technically probably look up their username, but it's handy to be able to see it. Once we're in a database, we'll likely have a separate table with the user_id to username translation, but for now it's easiest to have it in one place.